tests/dummy/app/templates/snippets/buttons-dropdown.hbs

```diff
@@ -0,0 +1,9 @@
+{{md-btn-dropdown text='Button' activates="dropdown-target"}}
```

Comment:
(something like) this could be helpful https://coderwall.com/p/qkk2zq/components-with-structured-markup-in-ember-js-v1-10

what do you think?